src/framework/db/storage.clj

```diff
@@ -1,9 +1,11 @@
 (ns framework.db.storage
   (:require
+    [com.stuartsierra.component :as component]
     [next.jdbc :as jdbc]))
```

Comment:
Maybe you can separate the database integration of the ALC solution, this one is an example https://github.com/GorillaStack/acl